src/api/private/notes/notes.controller.ts

```diff
@@ -174,6 +178,82 @@ export class NotesController {
     }
   }
 
+  @Post(':noteIdOrAlias/metadata/alias/:newAlias')
+  async addAlias(
+    @Req() req: Request,
+    @Param('noteIdOrAlias') noteIdOrAlias: string,
+    @Param('newAlias') newAlias: string,
+  ): Promise<NoteMetadataDto> {
```

Comment:
It's also very weird, that one operates (according to the path) on alias objects, but the response contains a whole metadata object...

Reply:
There can't be done much, at least not if we don't want't to split the NoteMetadataDto further down. Which I'm reluctant to do…